web/src/main/java/se/inera/intyg/webcert/web/csintegration/certificate/CreateCertificateFromCertificateService.java

```diff
@@ -62,6 +64,14 @@ private String createCertificate(String patientId, CertificateModelIdDTO modelId
         );
 
         pdlLogService.logCreated(response);
+        monitoringLogService.logUtkastCreated(
+            response.getMetadata().getId(),
+            response.getMetadata().getType(),
+            response.getMetadata().getUnit().getUnitId(),
+            response.getMetadata().getIssuedBy().getPersonId(),
+            0
```

Comment:
Suggest to create a constant for the "0". Call it something like NO_PREFILL_ELEMENTS.